tests/ui/test_ui_tools.py

```diff
@@ -1640,3 +1645,33 @@ def test_text_content(self, mocker,
                 count_str)
         assert len(text[0]) == len(expected_text) == (width - 1)
         assert text[0] == expected_text
+
+
+class TestTopicButton:
+    @pytest.mark.parametrize('width, count, stream_id, title, stream_name', [
+        (8, 2, 86, 'topic1', 'Django'),
+        (9, 1, 14, 'topic2', 'GSoC'),
+        (25, 1000, 205, 'topic3', 'PTEST'),
+    ])
+    def test_init_calls_top_button(self, mocker, width, count, title,
+                                   stream_id, stream_name):
+        controller = mocker.Mock()
+        controller.model.stream_dict = {
```

Comment:
@neiljp Not for this. I test for the `__init__` part here which works better with different TopicButton creations (More cases) rather than having a single `topic_button` fixture which calls init once. 

For more info, the 10th commit - `models: Mark 'muted' topics as when displaying topic view.` does include a topic_button fixture.

Reply:
Re-reading this, It seems you are asking if there is a 'stream_dict' fixture. *Oops*

Not yet. It is in the BuddyPR PR?

Reply:
Yes, apologies. I think it may possibly be in the test-refactoring PR #387 which will fix #385.